### DISCUSSION TITLE
handling ESC button

### DISCUSSION THREAD

**Discussion by rkrueck** [Exahilosys/survey#22] [Q&A] [answered]
Do you have an example on how to handle (in this case ignore) the escape button press?

**Comment by Exahilosys** [comment, marked as answer]
Heyo! All widgets (and therefore, routines) have an `escapable` argument that can be set to `False` to ignore the escape button being pressed.

```py
message = survey.routines.input('Message:', escapable = False)
```

Feel free to close the discussion if this suffices for your needs.

**Comment by rkrueck** [comment]
Thank you. That works as expected. Keep up the good work.
